exasol_advanced_analytics_framework/interface/create_event_handler_udf.py

```diff
@@ -71,19 +71,23 @@ def run(self, ctx) -> None:
         return_query_view = None
         return_query = None
         final_result = {}
-        if result.status != "finished":
-            schema = self.exa.meta.script_schema
-            return_query_view, return_query = self._wrap_return_query(
-                iter_num, bucketfs_connection, schema, result.return_query)
-        else:
+        query_list = []
+        if result.is_finished:
```

Comment:
Here it is probably better to check the type via instance of, otherwise the mypy later throws errors when access the attributes of the subclass

Reply:
maybe it's better to remove class inheritance, it doesn't seem necessary after our last update.

Reply:
yes, but the check of type is still necessary for mypi